zipkin-autoconfigure/ui/src/main/java/zipkin/autoconfigure/ui/ZipkinUiAutoConfiguration.java

```diff
@@ -59,6 +60,7 @@
  */
 @Configuration
 @EnableConfigurationProperties(ZipkinUiProperties.class)
+@ConditionalOnProperty(value = "zipkin.ui.enabled", havingValue = "true", matchIfMissing = true)
```

Comment:
Nits: `name` might be more clear than `value` (which is an alias for `name`). Maybe it is just me but `value = x, havingValue = y` reads kind of funny. Also, the default of `havingValue` should be sufficient for our purposes. Specifying `"true"` might mean something like `"TRUE"` doesn't work.